### DISCUSSION TITLE
nmrXiv Data Submission Workflow.

### DISCUSSION THREAD

**Discussion by NishaSharma14** [NFDI4Chem/nmrxiv#144] [Features]
Step-1:  **FILE UPLOAD**

![image](https://user-images.githubusercontent.com/54287612/170287200-b108c5c9-9ac0-4b06-a9fb-fe65d5a22428.png)

Step-2: **SIMILARITY SEARCH AND SPECTRA - ATOM ASSIGNMENTS**

![image](https://user-images.githubusercontent.com/54287612/170287484-500ee1ed-0629-4cb7-b09c-5df48e2602fa.png)

Step-3: **META DATA (Minimum information requirements / validations will be implemented at this stage )**

![image](https://user-images.githubusercontent.com/54287612/170287660-fe006de7-6008-45ea-853d-659a25e1e5d3.png)

Step-4: **COMPLETE**

![image](https://user-images.githubusercontent.com/54287612/170287826-1277d7d3-4daf-4eb4-947b-098c36925215.png)

**Comment by stefhk3** [comment]
Do we want to have an option for submissions without raw data?

**Reply by NishaSharma14** [reply]
As of now, it needs raw data and we are only supporting this. Without raw data, Minimum Information must be required, and also there must be proper validation to validate the source of information, otherwise, it might blot the repository with lot many unnecessary deposits.
But we can always discuss this and give a thought about its use cases & utilities.
